src/python/pants_release/generate_github_workflows.py

```diff
@@ -569,7 +569,8 @@ def upload_test_reports(self) -> Step:
                 f"""\
                 export S3_DST={s3_dst}
                 echo "Uploading test reports to ${{S3_DST}}"
-                ./src/python/pants_release/copy_to_s3.py \
+                ./pants run ./src/python/pants_release/copy_to_s3.py \
```

Comment:
Noticed this one here in the CI run for this PR: https://github.com/pantsbuild/pants/actions/runs/5426832979/jobs/9869613454#step:9:25

Reply:
The error is at least new: https://github.com/pantsbuild/pants/actions/runs/5427069488/jobs/9869957563#step:12:33

I'm being lazy - hopefully this is just expected on a branch.

Reply:
This did work on main.